server/src/test/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutorTest.java

```diff
@@ -149,7 +148,7 @@ private static void verifyTwoIndependentCommands(Replica replica,
                                                      Command<?> command1,
                                                      Command<?> command2) {
         final AtomicReference<Command<?>> lastCommand = new AtomicReference<>();
-        verify(replica.delegate, timeout(Duration.ofSeconds(2)).times(1)).apply(argThat(c -> {
+        verify(replica.delegate, timeout(TimeUnit.SECONDS.toMillis(2)).times(1)).apply(argThat(c -> {
```

Comment:
Nice 👍 
I found `timeout(Duration)` has been reverted at [3.2.3](https://github.com/mockito/mockito/releases/tag/v3.2.3) due to Android compatibility issue.
https://github.com/mockito/mockito/issues/1843